docs/dev/libvirt/README.md

```diff
@@ -208,6 +218,8 @@ openshift-install destroy cluster
 
 You can also use [`virsh-cleanup.sh`](../../../scripts/maintenance/virsh-cleanup.sh), but note that it will currently destroy *all* libvirt resources.
 
+Note that this might leave some unused images in your disk. To remove them and restore some disk space, clean the `/var/lib/libvirt/openshift-images` folder.
```

Comment:
destroy cluster should cleanup the cluster correctly. This script is only a Dev backstop.

Maybe add the disk cleanup?